packages/@aws-cdk/aws-ecs/README.md

```diff
@@ -237,6 +237,14 @@ const container = ec2TaskDefinition.addContainer("WebContainer", {
 
 You can specify container properties when you add them to the task definition, or with various methods, e.g.:
 
+```ts
+taskDefinition.addContainer("WebContainer", {
+  image: ecs.ContainerImage.fromRegistry("amazon/amazon-ecs-sample"),
+  memoryLimitMiB: 1024,
+  portMappings: [{ containerPort: 3000 }]
+})
+```
+
 ```ts
 container.addPortMappings({
   containerPort: 3000
```

Comment:
Can you add a comment above both to explain that there are 2 ways of defining the port mapping?

Reply:
Oh sure. Just saw this. Let me get right on that.